skimage/future/graph/rag.py

```diff
@@ -429,30 +426,28 @@ def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
     rag : RAG
         The Region Adjacency Graph.
     img : ndarray, shape (M, N, 3)
-        Input image.
+        Input image. If `colormap` is `None`, the image should be in RGB
+        format.
     border_color : colorspec, optional
-        Any matplotlib colorspec.
-    node_color : colorspec, optional
-        Any matplotlib colorspec. Yellow by default.
-    edge_color : colorspec, optional
-        Any matplotlib colorspec. Green by default.
-    colormap : colormap, optional
-        Any matplotlib colormap. If specified the edges are colormapped with
-        the specified color map.
-    thresh : float, optional
-        Edges with weight below `thresh` are not drawn, or considered for color
-        mapping.
-    desaturate : bool, optional
-        Convert the image to grayscale before displaying. Particularly helps
-        visualization when using the `colormap` option.
+        Color with which the borders between regions are drawn.
+    edge_width : float, optional
+        The thickness with which the RAG edges are drawn.
+    edge_cmap : colormap, optional
+        Any matplotlib colormap with which the edges are drawn.
+    img_cmap : colormap, optional
+        Any matplotlib colormap with which the image is draw. If set to `None`
+        the image is drawn as it is.
     in_place : bool, optional
         If set, the RAG is modified in place. For each node `n` the function
         will set a new attribute ``rag.node[n]['centroid']``.
+    ax : matplotlib axes, optional
+        The axes to draw on.
```

Comment:
I'm reviewing this for the first time and this confused me.
Probably, as far as `ax` is `optonal`, it's worth to add a note on default behaviour.